MAINTAINERS.yml

```diff
@@ -3521,6 +3521,19 @@ Silabs SiM3U Platforms:
   description: >-
     SiM3U SoCs, dts files, and related drivers. Boards based on SiM3U SoCs.
 
+Gardena Platforms:
+  status: maintained
+  maintainers:
+    - rettichschnidi
+  collaborators:
+    - M1cha
```

Comment:
This user does not seem to be a project contributor.
Please file a "Contributor Nomination" issue.
Otherwise it is not possible to assign @M1cha to reviews, and so forth

Reply:
Sent him an email, waiting for a reply.